src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1215,6 +1222,10 @@ private function getErrorMessages(Exception $e)
                 'There are not enough products in stock',
                 'Admin.Notifications.Error'
             ),
+            NegativePaymentAmountException::class => $this->trans(
+                'The amount should be greater than 0.',
```

Comment:
```suggestion
                'The amount of a payment should be strictly positive.',
```

Suggestion @LouiseBonnard what do you think?

Reply:
It sounds a bit authoritarian. Why changing the wording, isn't it clear enough?

If you want to emphasize the mandatory side of it, I would write _Invalid value: the payment must be a positive amount._, what do you think?

Reply:
It's not the mandatory side that I wanted to emphasize, but more having a less "technical" message I just though "positive amount" is maybe more understandable than "greater to 0"

Reply:
I agree, thanks @jolelievre. I still suggest _Invalid value: the payment must be a positive amount._ in this case, it looks clearer.